src/index.ts

```diff
@@ -94,6 +94,15 @@ export class Transport {
 		}
 	}
 
+	/**
+	 * Closes the channel.
+	 */
+	public async closeChannel() {
```

Comment:
Added a new function, this should be added to CHANGELOG-LATEST.md

Reply:
This function has been removed in the last changes but a new parameter was included in connection.close() function, should we add this change to CHANGELOG?

Reply:
Yes, this should go in the CHANGELOG.